app/views/confirmation_mailer/comment.html.haml

```diff
@@ -9,7 +9,7 @@
 
 %p
   Your comment:
-  = @comment.text
+  = comment_as_html(@comment.text)
```

Comment:
This method, which will generate html (mostly <p> tags) is currently a child of a <p> element. This is going to generate invalid markup.

I actually think in this context the comment is a blockquote, as we're quoting their comment. It's semantically a different type of text to the rest of the text in this file.

I recon something like:

```
%p Your comment:

%blockquote= comment_as_html(@comment.text)
```

Or, it would be worth having a look at how it's inserted into alert emails.